pandas/io/common.py

```diff
@@ -92,6 +91,34 @@ def _is_url(url):
         return False
 
 
+def _urlopen(url, session=None):
+    compression = None
+    content_encoding = None
+    try:
+        import requests
+        if session:
+            if not isinstance(session, requests.sessions.Session):
+                raise ValueError(
+                    'Expected a requests.sessions.Session object, '
+                    'got {!r}'.format(session)
+                )
+            r = session.get(url)
+        else:
+            r = requests.get(url)
+        r.raise_for_status
```

Comment:
What does this line do?

Reply:
It will raise an exception if there is any issue reading the URL (any non-200 OK status).

Reply:
But don't you need to actually call the function?  i.e. `r.raise_for_status()` (with parentheses)

Reply:
Yep. I totally forgot that when copying-and-pasting :smile: